substra/cli/interface.py

```diff
@@ -565,6 +565,7 @@ def get(ctx, asset_name, asset_key, expand, output_format, config, profile, user
     # method must exist in sdk
     method = getattr(client, f'get_{asset_name.lower()}')
     res = method(asset_key)
+    res['profile'] = profile
```

Comment:
I would recommend to not add the profile to the results returned by the SDK / server. 

It is not part of the data we want to display to the user, it is more something related to how this should be displayed.

This could be added as an option the print command?

Reply:
I changed that in the last commit, that's better indeed! 🙂